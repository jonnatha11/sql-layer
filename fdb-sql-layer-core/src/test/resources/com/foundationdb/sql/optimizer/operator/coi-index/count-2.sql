SELECT COUNT(*) FROM items WHERE items.sku < '8888'
