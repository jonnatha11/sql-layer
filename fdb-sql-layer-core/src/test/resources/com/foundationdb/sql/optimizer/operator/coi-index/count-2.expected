PhysicalSelect[_SQL_COL_1:bigint]
  Project_Default(Field(0))
    Count_Default(*)
      IndexScan_Default(Index(items.sku), sku < '8888')